tests/test_orcid.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import pytest
+from isb_lib.authorization import orcid
+
+orcid_id = "000-0003-2109-7692"
+token = "b035c8e7-60ee-4400-a6d3-9b02ff9ff1b0"
```

Comment:
Let's try to avoid putting tokens in GH, instead load from an env var (or cli argument, etc) and make a note about how to create it.

Reply:
Removed